src/js/models/v2/aboutTheData/ReportingOverviewRow.js

```diff
@@ -19,6 +19,8 @@ const BaseAgencyRow = {
         this.certified = data.recent_publication_date_certified || false;
         this.tasTotals = data.tas_account_discrepancies_totals;
         this._federalTotal = data.federalTotal;
+        this.unlinkedContractAwards = data.unlinked_contract_award_count;
```

Comment:
are we handling no data for these two? I guess the api could be responding with 0 if there is no data